packages/ra-ui-materialui/src/input/AutocompleteInput.js

```diff
@@ -307,9 +317,14 @@ export class AutocompleteInput extends React.Component {
         const { containerProps, children } = options;
 
         return (
-            <Paper {...containerProps} square>
-                {children}
-            </Paper>
+            <Popper
+                {...containerProps}
```

Comment:
I think te `containerProps` should be passed to the `Paper` element, otherwise it's a BC break.